src/controllers/legacycontrollersettings.h

```diff
@@ -263,7 +271,7 @@ class LegacyControllerNumberSetting
                 m_defaultValue >= m_minValue && m_savedValue >= m_minValue &&
                 m_editedValue >= m_minValue && m_defaultValue <= m_maxValue &&
                 m_savedValue <= m_maxValue && m_editedValue <= m_maxValue &&
-                m_stepValue > 0 && m_stepValue < m_maxValue;
+                m_stepValue > 0 && valid_range(m_minValue, m_maxValue, m_stepValue);
```

Comment:
We may face the condition that the default "default" = 0 in case of a missing attribute is not valid. Do we want to improve that as well for the negative range?

Reply:
Added the usecase at the de-serialisation step.